katello/katello/hostname-change.rb

```diff
@@ -161,6 +161,16 @@ def get_default_proxy_id
       proxy["Id"]
     end
 
+    def custom_cert_options_check(scenario_answers)
+      if scenario_answers["certs"]["server_cert"] &&
+         scenario_answers["certs"]["server_key"] &&
+         scenario_answers["certs"]["server_cert_req"] &&
```

Comment:
The trailing ```&&``` is odd here. Technically it'll work but I'd rather get rid of it.

Reply:
+1 had something after it that I removed and I forgot to delete the "&&"